services/common/src/components/common/utils/actionlessNetworkCalls.js

```diff
@@ -0,0 +1,137 @@
+import { ENVIRONMENT } from "@mds/common";
```

Comment:
This file already exists at: services/common/src/redux/utils/actionlessNetworkCalls.js

We can just use that one 😁 

![image](https://github.com/bcgov/mds/assets/83598933/ecef3d82-c54b-4283-ab9e-18b51e726693)

Reply:
Removed.